### PR TITLE
Fix previous changes to the build system and some docs

### PR DESCRIPTION
**Pull request by nmnobre**
## Bug description
Documentation uses "**" to try to bolden words when it should be "+"
The build system changes made in #26725 fail in multi-application builds

## How to reproduce
Examine the affected documentation pages
Build a combined application

## Impact
Build issues have to be fixed to move forward 

Hey @GiudGiud,

I feel a bit stupid for never realising @moosebuild gives you a full MOOSE site so you can see your documentation changes before they're merged. It's brilliant! I've checked all the changes here, and they look grand. :)

The only other change is the fix for the header symlink rule in the makefiles, hopefully it works for you too!

Cheers,
-Nuno

### CODE REVIEW

**Review by dschwen — APPROVED** [review]
Makefile change looks fine to me.

**Review by GiudGiud — COMMENTED** [review]
@nmnobre thanks for the PR!
I ll review and check this locally when I come back from travel

File: framework/doc/content/source/auxkernels/AdvectiveFluxAux.md [hunk 1]
```diff
@@ -7,14 +7,14 @@
...
+This auxkernel supports CONSTANT and FIRST MONOMIAL AuxVariable types. The advection velocity is required with [!param](/AuxKernels/AdvectiveFluxAux/vel_x) for advection flux calculation. [!param](/Postprocessors/SideAdvectiveFluxIntegral/vel_y) and [!param](/Postprocessors/SideAdvectiveFluxIntegral/vel_z) are needed for 2D and 3D simulations. The advected quantity can be either a variable [!param](/AuxKernels/AdvectiveFluxAux/advected_variable) or material property [!param](/AuxKernels/AdvectiveFluxAux/advected_mat_prop).
 
 !alert warning
-The expression of the advective flux in this object is generic, as described, and may differ from the advective flux implemented in your physics implementation. If so, you may not use this object to compute the advective flux.
+The expression of the advective flux in this object is generic, as described, and may differ from the advective flux implemented in your physics implementation. If so, you may not wish to use this object to compute the advective flux.
```
Comment (GiudGiud) [inline framework/doc/content/source/auxkernels/AdvectiveFluxAux.md:17]:
```suggestion
The expression of the advective flux in this object is generic, as described, and may differ from the advective flux implemented in your physics implementation. If so, you may not use this object to compute the advective flux.
```
if it s going to give the wrong result dont use it

**Review by GiudGiud — COMMENTED** [review]
lots of good catches

File: framework/doc/content/source/auxkernels/AdvectiveFluxAux.md (hunk 1 above)
Comment (nmnobre) [reply framework/doc/content/source/auxkernels/AdvectiveFluxAux.md]: Makes sense, done. :)

**Review by GiudGiud — APPROVED** [review]

**Comment by GiudGiud** [comment]
Thanks for the patch!

**Pull request merged**